beekeeper-cleanup/src/test/java/com/expediagroup/beekeeper/cleanup/monitoring/BytesDeletedReporterTest.java

```diff
@@ -48,6 +49,7 @@
 @ContextConfiguration(classes = { TestApplication.class },
   loader = AnnotationConfigContextLoader.class)
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+@TestPropertySource(properties = {"spring.jpa.hibernate.ddl-auto=update"})
```

Comment:
Why is this needed here?

Reply:
if its not in here we get a very chatty WARN log; `GenerationTarget encountered exception accepting command : Error executing DDL "drop table housekeeping_metadata" via JDBC Statement`.

Reply:
I think it's fine to do it since this is just a test, but out of curiosity: what's the full error?

Reply:
It's a hibernate issue. Seems that the db that gets created is derby and not sql, and it tries to drop the table before creating it, but it doesnt exist so throws the warning. 
`WARN  org.hibernate.tool.schema.internal.ExceptionHandlerLoggedImpl:27 - GenerationTarget encountered exception accepting command : Error executing DDL "drop table housekeeping_metadata" via JDBC Statement
org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "drop table housekeeping_metadata" via JDBC Statement
        at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:67)`

Reply:
Interesting, because for Derby the default for `spring.jpa.hibernate.ddl-auto` should be `create-drop`, which should work. Maybe it's overwritten somewhere else.

Reply:
That's odd. Also, why is it referencing the `housekeeping_metadata` table?